packages/react-focus-management/src/hooks/useFocusFinders.ts

```diff
@@ -0,0 +1,16 @@
+import { useFocusManagementContext } from '../focusManagementContext';
+import { Types as AHTypes } from 'ability-helpers';
+
+export const useFocusFinders = () => {
+  const { ahInstance } = useFocusManagementContext();
+
+  const findAll = (...args: Parameters<AHTypes.FocusableAPI['findAll']>) => ahInstance.focusable.findAll(...args);
+  const findFirst = (...args: Parameters<AHTypes.FocusableAPI['findFirst']>) => ahInstance.focusable.findFirst(...args);
+  const findLast = (...args: Parameters<AHTypes.FocusableAPI['findLast']>) => ahInstance.focusable.findLast(...args);
```

Comment:
Option 1: if we do not change anything in args, we don't need these closures
Option 2: I would rather restrict these API to allow pass only element now

Reply:
I've narrowed down the args now, since the other args might not make sense yet in the current implementation -> keeping it dumb